circus-train-comparator/src/test/java/com/hotels/bdp/circustrain/comparator/hive/HiveDifferencesIntegrationTest.java

```diff
@@ -15,6 +15,7 @@
  */
 package com.hotels.bdp.circustrain.comparator.hive;
 
+import static com.hotels.bdp.circustrain.comparator.api.ComparatorType.SHORT_CIRCUIT;
```

Comment:
This should come at the bottom of the statics, same order as the non-static for java, javax, org, com, com.expedia, com.hotels etc. Does Intellij not have the feature to do consistent ordering like Eclipse?

Reply:
It does, it just seems to not like importing formatting settings from Eclipse